### PR TITLE
chore(release): Update to `0.2.3` sdk

### PR DESCRIPTION
**Pull request by bengreenier**
Updates to use the new `0.2.3` SDK 🥳 

Additionally, add small fixes for new naming

https://rainway.com/blog/2022/01/28/changelog-0001/

### CODE REVIEW

File: assets/index.ejs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en" style="height: 100%">
 
 <head>
-  <title>Rainway Demo App</title>
+  <title>Rainway Stream SDK Sandbox</title>
```
Comment (bengreenier) [inline assets/index.ejs:5]: Hmm we should probably keep the styling we have here on `master` - will update

File: src/sandbox.ts [hunk 2]
```diff
@@ -103,15 +115,15 @@ class StreamWidget {
...
-        // Populate hostname from config, then localStorage, and finally default to "608f3a6b-0100-0000-0000-000000000000"
-        this.hostname.value =
-            config.hostname ??
-            localStorage.getItem("hostname-" + widgetClassName) ??
-            "000000000000000000";
+        // Populate peerId from config, then localStorage, and finally default to "555555555123456789"
+        this.peerId.value =
+            config.peerId ??
+            localStorage.getItem("peerId-" + widgetClassName) ??
+            "555555555123456789";
```
Comment (bengreenier) [inline src/sandbox.ts:122]: this should stay at zero

File: src/sandbox.ts [hunk 3]
```diff
@@ -296,35 +343,64 @@ export class StreamSandbox {
...
+    async connectToGateway(): Promise<void> {
         // Make and attach the Rainway client.
-        this.setUIState(SandboxState.ConnectingToRelay);
+        this.setUIState(SandboxState.ConnectingToGateway);
         if (!this.runtime) {
             this.runtime = await RainwayRuntime.initialize({
                 apiKey: sandboxApiKey,
-                externalId: "web-sdk-demo-sandbox",
-                // Listener for instant relay connection loss
+                externalId: "web runtime sandbox",
```
Comment (bengreenier) [inline src/sandbox.ts:353]: this should also stay

**Comment by bengreenier** [comment]
Tested locally - was able to:
- Connect/Disconnect from Gateway
- Connect/Disconnect from Stream
- Send messages
- Send input
- View video

**Review by lynn — APPROVED** [review]

**Comment by bengreenier** [comment]
hosted version updated as well. https://sdk-builds.rainway.com/demos/web/

**Pull request merged**